notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<h3>Create a feature store </h3><p> </p><p>Running the following code cell creates a feature store and prints the process logs. The method to create a feature store returns a <a href="https://google.aip.dev/151" target="_blank">long-running operation</a>&nbsp;(LRO), which is an asynchronous job. Other methods like updating and deleting a feature store are also asynchronous jobs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>